diff --git a/dist/index.cjs b/dist/index.cjs
index 3ae762beeb9b98bb4bf00c4306cbde2cd721f4e3..6d84038cf7f488a2818bd8a02ed2e175fa59528d 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -143,7 +143,7 @@ function viteReact(opts = {}) {
   const jsxImportDevRuntime = `${jsxImportSource}/jsx-dev-runtime`;
   let isProduction = true;
   let projectRoot = process.cwd();
-  let skipFastRefresh = false;
+  let skipFastRefresh = isProduction;
   let runPluginOverrides;
   let staticBabelOptions;
   const importReactRE = /\bimport\s+(?:\*\s+as\s+)?React\b/;
diff --git a/dist/index.mjs b/dist/index.mjs
index 5d06d5fa7ef5b4bdfa41ebe6bb780e46eb035d08..d68b55bcc3d8d64b049615e7f6e83f8476bdb592 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -127,7 +127,7 @@ function viteReact(opts = {}) {
   const jsxImportDevRuntime = `${jsxImportSource}/jsx-dev-runtime`;
   let isProduction = true;
   let projectRoot = process.cwd();
-  let skipFastRefresh = false;
+  let skipFastRefresh = isProduction;
   let runPluginOverrides;
   let staticBabelOptions;
   const importReactRE = /\bimport\s+(?:\*\s+as\s+)?React\b/;
